.github/workflows/CI.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags: '*'
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1'
+#          - 'nightly'
```

Comment:
is this deliberate?

Reply:
Some people complained about GitHub Actions not providing the equivalent `allow-failures`.

Reply:
I see. The current workaround to this is to disable `fail-fast`

https://github.com/JuliaImages/ImageCore.jl/blob/a90b8e4314228adcf147bee03cd0258915a30e61/.github/workflows/UnitTest.yml#L16-L17

Reply:
I'm happy to add it back. Will get to it in a bit.